---
title: Publications
type: archives
# The Archives of posts.
# v2.0
# https://github.com/cotes2020/jekyll-theme-chirpy
# © 2017-2019 Cotes Chung
# MIT License
#Preprints #_BioRxiv_
---

<head>
  <link
    href="https://fonts.googleapis.com/css?family=Montserrat"
    rel="stylesheet"
  />
  <link rel="stylesheet" href="../../assets/css/main.css" />
</head>

### Citation statistics can be accessed using [Google Scholar](https://scholar.google.com/citations?hl=en&authuser=2&user=AnY36aMAAAAJ)

#### **Preprint**

* **Fast and small footprint hybrid HMM-HIFIGAN based system for speech synthesis in Indian Languages** <br>
Sudhanshu Srivastava, Anusha Prakash,  Jom Kuriakose, <u>Ishika Gupta</u>, Hema A. Murthy  
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://arxiv.org/pdf/2302.06227.pdf" style="all: unset; color: inherit">Paper</a>
</button> 

#### **Publications 2023**
* **Technology Pipeline for Large Scale Cross-Lingual Dubbing of Lecture Videos into Multiple Indian Languages** 
<br>
Arun Kumar, Anusha P, Ashish Seth, Bhagyashree M, <u>Ishika Gupta</u>, Hema A Murthy, Mano R Kumar M et.al
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://arxiv.org/pdf/2211.01338.pdf" style="all: unset; color: inherit">Paper</a>
</button> 
<button type="button" class="btn btn-outline-secondary btn-sm">
<a href="https://docs.google.com/presentation/d/1rNp-4YC76Zd3qIiEVlfbYkwgkhWs9atESojxAL-TlSA/edit?usp=sharing" style="all: unset; color: inherit">Poster</a>
</button>

* **E-TTS: Expressive Text-to-Speech Synthesis for Hindi Using Data Augmentation** <br>
<u>Ishika Gupta</u>, Hema A. Murthy  
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://dl.acm.org/doi/10.1007/978-3-031-48312-7_20" style="all: unset; color: inherit">Paper</a>
</button> 
<button type="button" class="btn btn-outline-secondary btn-sm">
<a href="https://docs.google.com/presentation/d/1XSBO8trOrsBD1208HSKg4j2v698ptcsUsXJOLW-c3JM/edit?usp=sharing" style="all: unset; color: inherit">Poster</a>
</button>

<!-- #### **2023** -->

<!-- 
* **Long-read mapping to repetitive reference sequences using Winnowmap2** <br>
<u>C Jain</u>, A Rhie, N Hansen, S Koren, A Phillippy. _Nature Methods_
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://drive.google.com/file/d/1YxLym5_TfTRphj139xkIvI5Xw_o5P67s/view?usp=sharing" style="all: unset; color: inherit">Paper</a>
</button> 
<button type="button" class="btn btn-outline-secondary btn-sm">
<a href="https://github.com/marbl/Winnowmap" style="all: unset; color: inherit">Code</a>
</button> 
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://drive.google.com/file/d/1CwZ2RfgYGr2o8qXvfyXmpADf1he_xD9j/view?usp=sharing" style="all: unset; color: inherit">Slides</a>
</button>
<button type="button" class="btn btn-outline-success btn-sm"><a href="https://timesofindia.indiatimes.com/india/first-truly-complete-human-genome-sequenced-india-software-plays-key-role/articleshow/83364939.cms" style="all: unset; color: inherit">News</a></button> 

* **Accelerating minimap2 for long-read sequencing applications on modern CPUs** <br>
S Kalikar, <u>C Jain</u>, V Md, S Misra. _Nature Computational Science_
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://drive.google.com/file/d/1Xl3m6CX7lOdhEj1pq6VAhsJC-BeqTwDI/view?usp=sharing" style="all: unset; color: inherit">Paper</a>
</button> 
<button type="button" class="btn btn-outline-secondary btn-sm">
<a href="https://github.com/bwa-mem2/mm2-fast" style="all: unset; color: inherit">Code</a>
</button>

* **Co-linear chaining with overlaps and gap costs** <br>
<u>C Jain</u>, D Gibney, SV Thankachan. _JCB (extended abstract appeared in RECOMB'22)_
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://drive.google.com/file/d/10B0WIXyIawZTpcLDTAo3fo0Q35_8IKBI/view?usp=sharing" style="all: unset; color: inherit">Paper</a>
</button> 
<button type="button" class="btn btn-outline-secondary btn-sm">
<a href="https://github.com/AT-CG/ChainX" style="all: unset; color: inherit">Code</a>
</button> 
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://drive.google.com/file/d/1CWtaCIFn33strERXJF4qkNq9i-rTF9t1/view?usp=sharing" style="all: unset; color: inherit">Slides</a>
</button>

* **The complete sequence of a human genome** <br>
Nurk et al. _Science_
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://doi.org/10.1126/science.abj6987" style="all: unset; color: inherit">Paper</a>
</button>

* **Chasing perfection: validation and polishing strategies for telomere-to-telomere genome assemblies** <br>
Mc Cartney et al. _Nature Methods_
<br>
<button type="button" class="btn btn-outline-info btn-sm">
<a href="https://doi.org/10.1038/s41592-022-01440-3" style="all: unset; color: inherit">Paper</a>
</button> -->


<!-- #### **2020** -->
<!-- 
* **Weighted minimizer sampling improves long read mapping** <br>
<u>C Jain</u>, A Rhie, H Zhang, C Chu, B Walenz, S Koren, A Phillippy. _Bioinformatics (ISMB proceedings)_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1VNdlFJ1w36tUWjSprsatGyZs_KJugxD0/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/marbl/Winnowmap" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1Er4zEYOPqGz9V5b_h7073SBlIQMJn7jl/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>

* **A comprehensive evaluation of long read error correction methods** <br>
H Zhang, <u>C Jain</u>, S Aluru. _BMC Genomics_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-07227-0" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/haowenz/LRECE" style="all: unset; color: inherit">Code</a></button> 

* **On the complexity of sequence to graph alignment** <br>
<u>C Jain</u>, ⇔ H Zhang, Yu Gao, S Aluru. _JCB (extended abstract appeared in RECOMB'19)_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/12SN5wWBLv36frBzVvChoFSVx3by32t7t/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/haowenz/SGA" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1VjwSNp1Ir9lRQeV3H6JVJ1Sq3b_bGvy-/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>

#### **2019**

* **Strain-level metagenomic assignment and compositional estimation for long reads with MetaMaps** <br>
A Dilthey, <u>C Jain</u>, S Koren, A Phillippy. _Nature Comm._
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1D7ACvwyOwPusfNcyvz1xyTznZIoHHHo4/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/DiltheyLab/MetaMaps" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1pnLSgPaL-Y9LaDRphxcbUFk6Bs_38wo8/view?usp=sharing" style="all: unset; color: inherit">Poster</a></button>

* **Validating paired-end read alignments in sequence graphs** <br>
<u>C Jain</u>, H Zhang, A Dilthey, S Aluru. _WABI_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1ESLglHaG3Sl82iNx442KBPpQ-yEZGetB/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://drive.google.com/file/d/1AwevKNDGQpoOq5lz_1leh1VZFV4w3Dj_/view?usp=sharing" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://github.com/ParBLiSS/PairG" style="all: unset; color: inherit">Slides</a></button>

* **Accelerating sequence alignment to graphs** <br>
<u>C Jain</u>, S Misra, H Zhang, A Dilthey, S Aluru. _IEEE IPDPS_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1UHM97Tult4wOv6pXa3OowP6fpdMkCvkF/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/ParBLiSS/PaSGAL" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1nZj5l9EKpRMjgl1UcXI6WeGB6YrgVMTz/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>
<button type="button" class="btn btn-outline-success btn-sm"><a href="https://www.cc.gatech.edu/news/621751/identifying-differences-disease-causing-genomes-just-went-few-days-few-hours" style="all: unset; color: inherit">News</a></button> 

#### **2018**

* **High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries** <br>
<u>C Jain</u>, L Rodriguez, A Phillippy, K Konstantinidis, S Aluru. _Nature Comm._  
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1u6j7PN0QZPRMfaIr6GtU7Hxu5yVR3MgA/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/ParBLiSS/FastANI" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1kYS13Yu41llbM7QEbDBdLD094PtpqOPW/view?usp=sharing" style="all: unset; color: inherit">Poster</a></button>
<button type="button" class="btn btn-outline-success btn-sm"><a href="https://www.cc.gatech.edu/news/616894/new-approach-speeds-genomic-testing-microbial-species" style="all: unset; color: inherit">News</a></button> 

* **A fast adaptive algorithm for computing whole-genome homology maps** <br>
<u>C Jain</u>, S Koren, A Dilthey, A Phillippy, S Aluru. _Bioinformatics (ECCB proceedings)_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1YqY5fa7t3CpR5nN3xU0ntSjnlxR1N0B7/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/marbl/MashMap" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1gcBJL64B0Mjj2NlmtBUsz2fzAr1vZsdJ/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>

* **A fast approximate algorithm for mapping long reads to large reference databases** <br>
<u>C Jain</u>, A Dilthey, S Koren, S Aluru, A Phillippy. _JCB (extended abstract appeared in RECOMB'17)_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1r23RXyXvpSS0_Vcus-Wo747ARHcc0zi1/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/marbl/MashMap" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1COk7F6bdjklih1k5uR7FIaZeRqhPA3g6/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>

#### **2017**

* **An adaptive parallel algorithm for computing connected components** <br>
<u>C Jain</u>, P Flick, T Pan, O Green, S Aluru. _IEEE TPDS (extended abstract appeared in SC'15)_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1IzXwgbpzIVsJ3v0T2EjfIwTZQraeVvSw/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/ParBLiSS/parconnect" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1_fzz9losNp8SF_dXTnfOGm0l6XuDcekK/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>
<button type="button" class="btn btn-outline-success btn-sm"><a href="https://cacm.acm.org/blogs/blog-cacm/200433-sc16-adds-computational-reproducibility-to-student-competition/fulltext" style="all: unset; color: inherit">News</a></button> 

* **Efficient detection of viral transmissions with next-generation sequencing data** <br>
Rytsareva et al. _BMC Genomics_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3732-4" style="all: unset; color: inherit">Paper</a></button> 

* **Kmerind: A parallel library for k-mer indexing of biological sequences on distributed memory systems** <br>
T Pan, P Flick, <u>C Jain</u>, Y Liu, S Aluru. _IEEE TCBB (extended abstract appeared in ACM BCB'16)_  
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1tKBuB25M0qci-Za6NtAaqAwgy7ZriGX7/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-secondary btn-sm"><a href="https://github.com/ParBLiSS/kmerind" style="all: unset; color: inherit">Code</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1_ALtEMWnr5cf_NR3ey1PpYE7-b-MiKV_/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button>

* **Critical Assessment of Metagenome Interpretation- a benchmark of metagenomics software** <br>
Sczyrba et al. _Nature Methods_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://www.nature.com/articles/nmeth.4458" style="all: unset; color: inherit">Paper</a></button> 


#### **2016**

* **A self-correcting connected components algorithm** <br>
P Sao, O Green, <u>C Jain</u>, R Vuduc. _ACM FTXS_
<br>
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://dl.acm.org/doi/10.1145/2909428.2909435" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/11BGzNOdLtxFehVymvfs__Vwie_LYK0ni/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button> --> 

<!-- #### **2014** -->

<!--* **Fine-grained GPU parallelization of pairwise local sequence alignment** <br>
<u>C Jain</u>, S Kumar. _IEEE HiPC_
<br>
 <button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/1NKVgF0wI73DwxLldSpSGj5_BAknA2X_S/view?usp=sharing" style="all: unset; color: inherit">Paper</a></button> 
<button type="button" class="btn btn-outline-info btn-sm"><a href="https://drive.google.com/file/d/11iYwjh0r-sXCSGsLrUm689imAsdz5qb4/view?usp=sharing" style="all: unset; color: inherit">Slides</a></button> --> 
